ROS2 Rust workshop devcontainer.md

```diff
@@ -0,0 +1,50 @@
+# ROS2 Rust workshop using DevContainer
```

Comment:
Modifying the ipython notebook was a bit too much work, and I don't know how you would like to structure it.
If you want to support both manual docker workflow and devcontainers, you'll need two sets of instructions I guess.

Reply:
This is a good point, at the beginning there were two build explanations, one for the devcontainer and the other one for the docker, but I removed the devcontainer one as I thought that it was better to focus on only the docker one to make everyone work in the same way. But I can add again now. Thank you!

Reply:
To support two workflows from the same Dockerfile I would recommend to make the manual scripts to use the same paths than the devcontainer (and over time you may make people move to use devcontainers, they are very convenient).

I usually use docker targets (e.g. `FROM image AS build` `FROM build AS develop` `FROM development AS deploy`) and you call your target on doing docker build, in case there are differences in the different workflows (allows you to have a base image, an image with development tools, and an image that has the bare minimum for deployment, for example).

But I'm digressing here

Reply:
Wow, that's class! https://docs.docker.com/build/building/multi-stage/ I will create a base docker image that contains everything needed just using docker, and then call in the devcontainers file. Will be much helpful as we won't need the use of the build_image.sh neither the use of the run_user.sh.  Simpler is always the better. Thank you!!!